### ISSUE TITLE
How to dump FLAME paramters from fitting results?

### ISSUE THREAD

**Issue by 199ChenNuo** [TimoBolkart/TF_FLAME#61]
I get a `*.ply` output from `fit_3D_mesh.py`, and I want to get FLAME parameters directly from the output or during fitting.
I output the `tf_pose` and `tf_exp` in the function `fit_3D_mesh()`, from the source code I know that
```
tf_pose = [neck_pose, jaw_pose, eyeballs_pose]
```
And is the `tf_exp` the FLAME expression parameters?

I also wonder if there is a direct way to generate FLAME model from parameters so that I can test whether I got the right FLAME parameters. I noticed that there is an online viewer provided on the FLAME website, but I didn't find the relevant code in the [Mesh](https://github.com/MPI-IS/mesh) project.

**Comment by TimoBolkart** [comment]
Yes, tf_exp are the expression parameters. The Mesh package has a MeshViewer to visualize meshes, otherwise one can output the FLAME mesh and check it in MeshLab
